polkadot/zombienet_tests/functional/0007-dispute-freshly-finalized.zndsl

```diff
@@ -21,9 +21,9 @@ honest: reports polkadot_parachain_candidate_dispute_concluded{validity="valid"}
 honest: reports polkadot_parachain_candidate_dispute_concluded{validity="invalid"} is 0 within 100 seconds
 
 # Check lag - approval
-honest: reports polkadot_parachain_approval_checking_finality_lag is 0
+honest: reports polkadot_parachain_approval_checking_finality_lag is lower than 2
 
 # Check lag - dispute conclusion
-honest: reports polkadot_parachain_disputes_finality_lag is 0
+honest: reports polkadot_parachain_disputes_finality_lag is lower than 2
```

Comment:
was this metric also lagging?

Reply:
Yes, it was let me figure it why.

Reply:
Yeah, that's expected with the way `polkadot_parachain_disputes_finality_lag`, because DetermineUndisputedChain https://github.com/paritytech/polkadot-sdk/blob/809d46657f588ff938ca2e9fa1208e7ff21f33fe/polkadot/node/service/src/relay_chain_selection.rs#L515 is called with the list of blocks that are already approved by approval-voting, so when we compute the disputes_finality_lag we already include in it the approval-voting-lag because  the subchain number can't be higher than the  highest approved block in `approval-voting`: 
https://github.com/paritytech/polkadot-sdk/blob/809d46657f588ff938ca2e9fa1208e7ff21f33fe/polkadot/node/service/src/relay_chain_selection.rs#L530